nightly/pytest-contracts.txt

```diff
@@ -1,6 +1,8 @@
 # python tests for smart contract deployment and invocation
 pytest contracts/deploy_call_smart_contract.py
 pytest contracts/deploy_call_smart_contract.py --features nightly
+pytest contracts/deploy_call_global_smart_contract.py
```

Comment:
global contract are not part of the stable protocol version yet so this will fail